client/my-sites/no-results/style.scss

```diff
@@ -4,6 +4,22 @@
 	color: var(--color-neutral-50);
 	font-size: $font-body;
 	line-height: 56px;
+
+	.no-results__titles {
+		margin-bottom: 4px; // +20 from elements = 24 gap
+		color: var(--color-text);
+		line-height: 1.5;
+
+		.no-results__title {
+			@extend .wp-brand-font;
+			font-size: $font-title-medium;
+			font-weight: 500;
+		}
+
+		.no-results__subtitle {
+			font-size: $font-body-small;
+		}
+	}
```

Comment:
I think we could align the title with the results below.

![align](https://user-images.githubusercontent.com/6586048/197734106-a691d415-1910-46fa-886b-32843dc0e2e4.png)

Reply:
Hey @vinimotaa, what do you think about it?